.github/dependabot.yml

```
@@ -29,6 +29,8 @@ updates:
   groups:
     kubernetes:
       patterns: ["k8s.io/*"]
+    capi:
+      patterns: ["sigs.k8s.io/cluster-api", "sigs.k8s.io/cluster-api/test"]
```

This good already, we can also do the following as k8s above 
```suggestion
      patterns: ["sigs.k8s.io/cluster-api/*]
```

---

Does this actually work, as the cluster-api dep does not include trailing slash. so it does not match?

```
    sigs.k8s.io/cluster-api v1.8.3
```

---

You have a point. I am not sure TBH. Let’s merge it like this